src/gameplay/tests/sinad.test.ts

```diff
@@ -0,0 +1,192 @@
+import  { GameConfig, isSubsetOf }  from '../game';
+import Game from '../game';
+import { RoomConfig } from '../room';
+import { GameMode } from '../gameModes';
+import { ReadyPrompt } from '../../sockets/readyPrompt';
+import { RoomCreationType } from '../roomTypes';
+
+
+jest.mock('../gameWrapper');
+
+
+const vh = {
+  'Piplup':
+    [
+      [
+        'VHpicked VHreject',
+        'VHapprove',
+      ],
+      [
+        'VHapprove',
+      ],
+      [
+        'VHleader VHapprove',
+      ],
+    ],
+  'Torchic':
+    [
+      [
+        'VHpicked VHleader VHreject',
+        'VHapprove',
+      ],
+      [
+        'VHapprove',
+      ],
+      [
+        'VHapprove',
+      ],
+    ],
+  'Mudkip':
+    [
+      [
+        'VHreject',
+        'VHpicked VHapprove',
+      ],
+      [
+        'VHpicked VHapprove',
+      ],
+      [
+        'VHpicked VHapprove',
+      ],
+    ],
+  'Turtwig':
+    [
+      [
+        'VHreject',
+        'VHleader VHapprove',
+      ],
+      [
+        'VHapprove',
+      ],
+      [
+        'VHpicked VHapprove',
+      ],
+    ],
+  'MewTwo':
+    [
+      [
+        'VHreject',
+        'VHapprove',
+      ],
+      [
+        'VHpicked VHleader VHapprove',
+      ],
+      [
+        'VHpicked VHapprove',
+      ],
+    ],
+  'Charmander':
+    [
+      [
+        'VHreject',
+        'VHpicked VHapprove',
+      ],
+      [
+        'VHpicked VHapprove',
+      ],
+      [
+        'VHpicked VHapprove',
+      ],
+    ],
+};
+
+const roomConfig: RoomConfig = new RoomConfig(
+  '1',
+  1,
+  jest.fn(),
+  10,
+  '', //password
+  GameMode.AVALON,
+  false,
+  new ReadyPrompt(),
+);
+const gameConfig: GameConfig = new GameConfig(
+  roomConfig,
+  false,
+  false,
+  false,
+  RoomCreationType.CUSTOM_ROOM,
+  jest.fn(),
+);
+
+const mockGame: Game = new Game(gameConfig);
+
+
+const adjustGame = (game: Game) => {
+ 
+      game.voteHistory = vh; 
+      game.enableSinadMode = true;
+      game.playersInGame = ['Piplup', 'Torchic', 'Mudkip', 'Turtwig', 'MewTwo', 'Charmander'];
+      game.gameMode = GameMode.AVALON;
+      game.missionHistory = ['succeeded','succeeded','failed'];
```

Comment:
Build error:
```
Property 'voteHistory' does not exist on type 'Game'.
Property 'enableSinadMode' does not exist on type 'Game'.
```
Not sure why this would happen, since the other fields (`missionHistory`, `playersInGame`) did not throw this error.